mscore/events.cpp

```diff
@@ -997,6 +997,7 @@ void ScoreView::contextMenuEvent(QContextMenuEvent* ev)
                   popup->addAction(getAction("edit-style"));
                   popup->addAction(getAction("page-settings"));
                   popup->addAction(getAction("load-style"));
+				  popup->addAction(getAction("edit-header-footer"));
```

Comment:
formatting, we use spaces (6 per level currently) not tabs.

Reply:
Sure will update that.